common/src/main/java/org/figuramc/figura/animation/Animation.java

```diff
@@ -607,5 +627,17 @@ public enum LoopMode {
         HOLD
     }
 
-    public record AnimationChannel(TransformType type, Keyframe... keyframes) {}
+    @LuaWhitelist
+    @LuaTypeDoc(
+            name = "AnimationChannel",
+            value = "animation_channel"
+    )
+    public record AnimationChannel(TransformType type, Keyframe... keyframes) {
```

Comment:
Oh, does `@LuaWhitelist` not work on record fields?

Reply:
uhhhh idk lol

Reply:
i jsut forgot to add it i think